client/src/app/(onboarding)/page.tsx

```diff
@@ -9,38 +9,72 @@ export default function Home() {
       <LandingPageHero />
       {/*Section 2 Block*/}
       <div className="sec2-container">
-      <div className="sec2-text-box">
-          <Typography variant="h3">Talk to other applicants</Typography>
+        <div className="sec2-text-box">
+          <Typography
+            variant="h2"
+            style={{
+              fontWeight: "bold",
+              textAlign: "center",
+              width: "700px",
+            }}
+          >
+            Talk to other applicants
+          </Typography>
 
-          <Typography variant="body1">
+          <Typography
+            variant="h4"
+            style={{
+              textAlign: "center",
+              width: "600px",
+            }}
+          >
             Engage with company thread to ask questions and support fellow
             applicants
           </Typography>
         </div>
-        <div className="sec2-image">
         <Image
           src="/LandingPage-images/rat_2.png"
```

Comment:
This Image is breaking the page. In the future, can you not push changes that obviously has errors on it? Thank you.
<img width="1339" alt="image" src="https://github.com/user-attachments/assets/992e9f78-9061-4e8b-bae5-cda2f7c3b475">